debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java

```diff
@@ -118,12 +118,17 @@ protected void validateConnection(Map<String, ConfigValue> configValues, Configu
                                 " FROM pg_catalog.pg_auth_members m" +
                                 " JOIN pg_catalog.pg_roles b ON (m.roleid = b.oid)" +
                                 " WHERE m.member = r.oid), 'rdsrepladmin') AS BOOL) IS TRUE AS aws_repladmin" +
+                                ", CAST(array_position(ARRAY(SELECT b.rolname" +
+                                " FROM pg_catalog.pg_auth_members m" +
+                                " JOIN pg_catalog.pg_roles b ON (m.roleid = b.oid)" +
+                                " WHERE m.member = r.oid), 'rds_replication') AS BOOL) IS TRUE AS aws_replication" +
                                 " FROM pg_roles r WHERE r.rolname = current_user",
                         connection.singleResultMapper(rs -> rs.getBoolean("rolcanlogin")
                                 && (rs.getBoolean("rolreplication")
                                         || rs.getBoolean("aws_superuser")
                                         || rs.getBoolean("aws_admin")
-                                        || rs.getBoolean("aws_repladmin")),
+                                        || rs.getBoolean("aws_repladmin")
+                                        || rs.getBoolean("aws_replication")),
```

Comment:
So in the Debezium docs it says "rds_replication"; which one is it actually?

Reply:
It is `rds_replication`, I just gave it a new name to keep the style the same as the rest. Please check lines 121-124.

Reply:
Ah yes, of course. Thanks for clarifying, it was late for me last night :)